### ISSUE TITLE
has this been tested with headless Factorio servers?

### ISSUE THREAD

**Issue by intcreator** [oorzkws/FactorioAchievementEnabler#16]
I run a headless modded Factorio server with a couple friends. will this repo be able to enable achievements there? I'm using the command

```
start /wait .\bin\x64\factorio.exe --start-server-load-latest
```

to start my server. if I put the DLL in the same folder as the executable should I append `/c start "" %command%` to the command I'm running? I tried this and it doesn't work. the achievements say that a console command was used which I don't remember doing (I definitely don't remember seeing the warning) so idk if this is what it always says for this repo's use case or if this repo doesn't apply to me anyway

for more context I'm running Factorio 1.1.109 with Space Exploration, Krastorio 2, and several QoL mods. I might have run a command in the server console without knowing it disabled achievements but now I'm 250+ hours in so I don't remember for sure

**Comment by oorzkws** [comment]
This would be run on clients, not the server. This will not enable achievements when:
- A console command has been used
- The player does not meet the hours threshold for the server to be eligible for achievements

Does this help? I don't see it causing the "a console command has been used" message.

**Comment by intcreator** [comment]
my bad, what I wrote was confusing. I've had the message that a console command was used before I tried using this repo. I just am not sure if Factorio says that a command was used even if achievements are actually only disabled my mods or there's a different message that says achievements have been disabled by mods.

are achievements recorded on the client side only? even though in theory one player could earn an achievement due to actions of a different player?
